### ISSUE TITLE
Magma - Spigot mod blockscan + Forge playerclaims ... DynmapCBBridge

### ISSUE THREAD

**Issue by cryofgaming** [webbukkit/dynmap#3673]
> template is **bold**
> sample data is *italicized*

**Issue Description:** Aight i have Dynmap-Forge installed, put the files von dynmapblockscanner into /renderdata and everything is working as expected, if i now disabled Dynmap-Forge, enable Dynmap-Spigot, but the dynmapblockscanner into /renderdata (Ofc i put the files in the correct plugin/dynamp/renderdata folder) Dynmap-Spigot wont load the files.

Is it possible that Forge-Spigot is just not able to show mod blocks?

* **Dynmap Spigot-version:** *v3.2.1-531*
* **Dynmap Forge-version:** *3.0.1-343*
* **Server Version:** *Magma 1.12.2*
* **Pastebin of Configuration.txt:** *both default*
* **Other Relevant Data/Screenshots:**

Dynmap-Forge mod processing log: (Everything working correctly)
![image](https://user-images.githubusercontent.com/96870153/153290797-03a41672-d692-43a4-afcf-278914ffb757.png)
... and so on

Dynmap-Spigot mod processing log: (here its not loading the renderdata files, although they are in the correct folder)
![image](https://user-images.githubusercontent.com/96870153/153290916-8c9d6316-d3ab-4aa6-9656-56fae57287be.png)

 [X] *I have looked at all other issues and this is not a duplicate*  
 [X] *I have been able to replicate this*

**Comment by mikeprimm-triva** [comment]
Dynmap-Spigot is not supported on modded servers (Fabric or Forge) - if you are using something that is Forge + Spigot API, it's probably not officially supported (we don't test on or support Magma, for example - if it works, it's a pleasant surprise, but not a goal...), and it'd be the Forge version that would have the best shot at working.

In this case, for example, the listing of the installed mods - which is not part of the Spigot API, since it doesn't recognize the notion of mods other than 'minecraft', results in us not knowing that there is any point to loading those rendering data...

**Comment by cryofgaming** [comment]
> Dynmap-Spigot is not supported on modded servers (Fabric or Forge) - if you are using something that is Forge + Spigot API, it's probably not officially supported (we don't test on or support Magma, for example - if it works, it's a pleasant surprise, but not a goal...), and it'd be the Forge version that would have the best shot at working.
> 
> In this case, for example, the listing of the installed mods - which is not part of the Spigot API, since it doesn't recognize the notion of mods other than 'minecraft', results in us not knowing that there is any point to loading those rendering data...

Ah yeah, I was afraid that this would be the case.

I have a Magma-server having Mods and few plugins, including “GriefPrevention” – Basically a plugin where you can create claims and such.
Dynmap should ofc render the mod blocks, but also show the claims for players… The problem is just to render the Blocks correctly, I need the Dynmap-Forge and to show the player-claims i need Dynmap-Spigot… Yea, you probably see the problem.

I saw there is some "DynmapCBBridge" (https://www.curseforge.com/minecraft/mc-mods/dynmapcbbridge). I guess fixing my issue, so Plugins can communicate with Dynmap-Forge.
But installing “DynmapCBBridge” into Plugins folder, and normally installing Dynmap-Forge basically does nothing. The Bridge gets shown in /plugins (Basically it shows “dynmap”) but it doesn't seem to work – claims won't be shown.

I guess now I get redirected to the DynmapCBBrige GitHub to post an issue there, but maybe you (or someone) has a solution for me.

And yes, I installed Grief Prevention Dynmap Support (https://www.spigotmc.org/resources/grief-prevention-dynmap-support.2881/) – When Dynmap-Spigot is active, it also works correctly.

Any ideas?

**Comment by mikeprimm** [comment]
Yep - DynmapCBBridge is kind of the thing for that - I haven't updated it for a while (it was there for Cauldron, which died out back in the 1.7.10 timeframe, IIRC), but it'd probably be easy enough to do a refresh!

**Comment by generrosity** [comment]
Just cleaning up the ticket - obviously this sits in the 'magma not supported' bucket for the moment. And sliding towards 'feature request: update CB Bridge plugin' 👍
